apps/web/components/v2/eventtype/EditLocationDialog.tsx

```diff
@@ -308,6 +310,15 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
             }}
           />
           {selectedLocation && LocationOptions}
+          <p className="text-sm text-gray-400">
+            <Trans i18nKey="cant_find_the_right_video_app_visit_our_app_store">
+              Cant find the right video app? Visit our
+              <LinkText href="/apps/categories/video" classNameChildren="text-blue-400" target="_blank">
+                App Store
+              </LinkText>
+              .
+            </Trans>
+          </p>
```

Comment:
From what I understand, this text only appears when the user clicks on the "+ Add a location" button, which opens up prompt to select from a list of locations (including video apps such as google and cal), this text makes sense when none of these options are selected (and the user can't find their preferred video app)

I believe, this could be a easy solution: (`!selection`)
```
{!selection && (
  <div className="mt-5">
    <p className="text-sm text-gray-400">
      <Trans i18nKey="cant_find_the_right_video_app_visit_our_app_store">
        Cant find the right video app? Visit our
        <LinkText href="/apps/categories/video" classNameChildren="text-blue-400" target="_blank">
          App Store
        </LinkText>
        .
      </Trans>
    </p>
  </div>
)}
```

I have created a small loom to demonstrate this:
https://www.loom.com/share/a8721373474842379c6a681900718c78

Reply:
Hey @SauravL3010 thank you for the review, this definitely makes sense. Will make the required changes